---
sensor_msgs/CameraInfo camera_info
